.gitignore

```diff
@@ -34,4 +34,4 @@ lerna-debug.log*
 /packages/server/dist
 /packages/client/dist
 
-/package-lock.json
+package-lock.json
```

Comment:
We actually do want to keep the `package-lock.json` files of the packages so should keep `/package-lock.json`